compiler/luci/pass/src/RemoveDuplicateTransposePass.cpp

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ * Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "luci/Pass/RemoveDuplicateTransposePass.h"
+
+#include <luci/IR/CircleNodes.h>
+#include <luci/IR/CircleOpcode.h>
+
+namespace luci
+{
+
+bool check_perm(luci::CircleConst *pred_perm, luci::CircleConst *main_perm)
```

Comment:
if you only read the arguments, the it would be better to use `const` like
```cxx
bool check_perm(const luci::CircleConst *pred_perm, const luci::CircleConst *main_perm)
```

Reply:
maybe add a comment what this `check_perm()` checks?

Reply:
Okay, I will fix it. and also, `check_perm()` with 2 permutation values will return true if where combination of  2 permutation values is [0,1,2,3......,n]. I will add this comment.

Reply:
Thanks for detail comment about it. I will change to `Return true if first_prem[second_prem[i]] == i`.